.changeset/beige-pumpkins-brush.md

```diff
@@ -0,0 +1,7 @@
+---
+"@justeattakeaway/pie-icons-react": minor
+"@justeattakeaway/pie-icons-vue": minor
+"@justeattakeaway/pie-icons": minor
+---
+
+[Fixed] - Sandwhich renamed to Sandwich (typo)
```

Comment:
I wonder if we could run a spell-check extension against the icon list in the docs site and find any more typos (or something smarter)

Reply:
Yeh, I don't know. The easiest way may be to spellcheck the index files created by the react/vue packages potentially, as they output all the names in code (although would have to do some normalisation of name prefixes to do that)